elastic/indexer/elastic-indexer.go

```diff
@@ -121,6 +127,23 @@ func (i *Indexer) runStep() error {
 	return nil
 }
 
+func (i *Indexer) checkAllResultMessagesAreValid(result *bulkResult) error {
+	for _, bulkResultOperation := range result.Items {
+		if bulkResultOperation.Index.ID != "" {
+			return checkHttpStatus(bulkResultOperation.Index.Status)
+		} else if bulkResultOperation.Update.ID != "" {
+			return checkHttpStatus(bulkResultOperation.Update.Status)
+		}
+	}
+}
+
+func checkHttpStatus(httpStatus int) error {
```

Comment:
I would expect `checkHttpStatus` to return a bool
What do you think about calling this `ErrorForStatus`?
That would make the calling code quite expressive

Reply:
`ErrorForHttpStatus` sounds good